src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ModuleBuilderImpl.cs

```diff
@@ -967,7 +967,7 @@ internal EntityHandle TryGetMethodHandle(MethodInfo method)
             arrayMethod.DeclaringType!.GetElementType() is TypeBuilderImpl tb && tb._handle == default;
 
         private static bool IsConstructedMethodFromNotBakedMethodBuilder(MethodInfo method) => method.IsConstructedGenericMethod &&
-            (method.GetGenericMethodDefinition() is MethodBuilderImpl mb && mb._handle == default || ContainsNotBakedTypeBuilder(method.GetGenericArguments()));
+            (method.GetGenericMethodDefinition() is MethodBuilderImpl mb && mb._handle == default || ContainsTypeBuilder(method.GetGenericArguments()));
```

Comment:
The name of this method `IsConstructedMethodFromNotBakedMethodBuilder` does not quite match what the implementation does.

Reply:
Right, and checking method handle here is also redundant, updated